src/webrtc.ts

```diff
@@ -668,18 +661,18 @@ export class WebrtcExtension extends EventEmitter implements StreamingClientExte
       });
   }
 
-  async _refreshIceServers (): Promise<ExtendedRTCIceServer[]> {
-    if (!this.stanzaInstance) {
+  async _refreshIceServers (stanzaInstance?: NamedAgent): Promise<ExtendedRTCIceServer[]> {
```

Comment:
we had to change a lot of this ice stuff because we need to configure/set the ice stuff up before we actually set the instance as the activeInstance.